Lib/test/regrtest.py

```diff
@@ -438,6 +438,8 @@ def main(tests=None, testdir=None, verbose=0, quiet=False,
 
     if slaveargs is not None:
         args, kwargs = json.loads(slaveargs)
```

Comment:
Do you want to backport the slaveargs changes just backported to 3.5? Or there is no need?